.github/workflows/wheels.yml

```diff
@@ -34,13 +41,13 @@ jobs:
         with:
           path: ./wheelhouse/*.whl
           name: wheels-${{ matrix.os }}
-  build_wheels_32bit:
-    name: Build wheels 32bit
+  build_wheels_macos_arm:
```

Comment:
I think it's worth adding a comment (or even changing the title) for this job, since it's only here to build the macOS Python 3.8 wheel for ARM, correct?

Reply:
Sure, I'll update the job name

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/12102/commits/a340b51b1be6e8727ec1ba1222a7764735142c10